tests/integration/docker/dns.bats

```diff
@@ -42,19 +44,37 @@ setup() {
                     break;
                 fi
         done
+	info "init swarm"
 	$DOCKER_EXE swarm init ${swarm_interface_arg}
-	$DOCKER_EXE service create --name testswarm1 --replicas $number_of_replicas --publish 8080:80 nginx /bin/bash -c "hostname > /usr/share/nginx/html/hostname; nginx -g \"daemon off;\""
-	$DOCKER_EXE service create --name testswarm2 --replicas $number_of_replicas --publish 8082:80 nginx /bin/bash -c "hostname > /usr/share/nginx/html/hostname; nginx -g \"daemon off;\""	
-	$DOCKER_EXE service create --name testdns --replicas $number_of_replicas --publish 8084:80 mcastelino/nettools sleep 60000
-	while :
-        do
-                total_tests=`$DOCKER_EXE ps --filter status=running --filter ancestor=nginx:latest --filter ancestor=mcastelino/nettools:latest | wc -l`
-                if [ $total_tests -lt $total_number_of_tests ]; then
-                    sleep 1
-                else
-                        break;
-                fi
-        done
+
+	info "create service testswarm1"
+	$DOCKER_EXE service create \
+	--name testswarm1 \
+	--replicas $number_of_replicas \
+	--publish 8080:80 nginx /bin/bash \
+	-c "hostname > /usr/share/nginx/html/hostname; nginx -g \"daemon off;\""
+
+	info "create service testswarm2"
+	$DOCKER_EXE service create \
+	--name testswarm2 \
+	--replicas $number_of_replicas \
+	--publish 8082:80 nginx /bin/bash \
+	-c "hostname > /usr/share/nginx/html/hostname; nginx -g \"daemon off;\""
+
+	info "create service testdns"
+	$DOCKER_EXE service create \
+	--name testdns \
+	--replicas $number_of_replicas \
+	--publish 8084:80 \
+	mcastelino/nettools sleep 60000
```

Comment:
@jodh-intel @grahamwhaley sure, taking a look to the code looks like that is the intention, but I   also prefer to do a blocking container anyway it will be destroyed at the end.
I would like @GabyCT input , I think she did this tests.